site/docs/guide/deployment-types.md

```diff
@@ -69,15 +68,17 @@ _Time for ice cream again!
 The employee already greets you with your first name by now.
 Asked about some ice cream of your favorite kind, the employee smiles at you and freezes.
 In fact, you don't get any response at all.
-So you decide to wait, firmly smiling back.
-And you wait.
+So you decide to wait, firmly smiling back._
+
```

Comment:
Why the newline? This belongs together, no? 🤔

If anything, I could imagine having this before *Some hours before the next sunrise*.